From 54fd7face86f3d5287bc8d0f2633bd13c66498cf Mon Sep 17 00:00:00 2001
From: Vitaly Lavrov <vel21ripn@gmail.com>
Date: Sun, 30 Sep 2018 22:49:18 +0300
Subject: [PATCH 5/9] Do not use the available clang as the default compiler.
 Use the CC environment variable when running autogen.sh and configure.

---
 configure.seed | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/configure.seed b/configure.seed
index 8f8817f..b150fdb 100644
--- a/configure.seed
+++ b/configure.seed
@@ -51,12 +51,6 @@ if test $SHORT_MACHINE = "arm"; then
 else
      AC_CHECK_LIB([numa], [numa_available], [LIBNUMA="-lnuma"])
 fi
-
-if test -z `which clang`; then
-CC=gcc
-else
-CC=clang
-fi
  
 HS_LIB=
 HS_INC=
-- 
2.19.1

